WorkInProgress/Administration/xASL_adm_RegisterAtlas.m

```diff
@@ -5,11 +5,19 @@
 % __________________________________
 
 %% Please add here the path to the atlas files and to ExploreASL
+
```

Comment:
I would move this function to our WorkInProgress folder, it is not a real function but a script depending on local paths. Useful script though, did you check that DARTEL works better than GS or oldnormalise for registering atlases?

Can't remember what I did previously exactly :)

BTW: are all regions mutually exclusive, then you can add them to a single volume

Reply:
It is already in WIP :)
Yes - it's IXI512, so we need DARTEL/GS instead of old-normalise. 
DARTEL likely does almost the same job as GS, so I'd say DARTEL is good.

In theory, we can merge the two atlases, they should be indeed mutually exclusive. But it has two different sources, so I'd prefer to keep them apart for this license/source reason. Technically, they can indeed be merged. What do you think?